rdr_service/workflow_management/nph/sms_workflows.py

```diff
@@ -3,14 +3,27 @@
 
 from protorpc import messages
 
+from rdr_service import config
 from rdr_service.api_util import open_cloud_file
 from rdr_service.dao.study_nph_sms_dao import SmsJobRunDao, SmsSampleDao, SmsN0Dao, SmsN1Mc1Dao
 from rdr_service.offline.sql_exporter import SqlExporter
 from rdr_service.workflow_management.general_job_controller import JobController
 from rdr_service.services.ancillary_studies.nph_incident import NphIncidentDao
+from rdr_service.services.slack_utils import SlackMessageHandler
 from rdr_service.workflow_management.nph.sms_validation import SmsValidator
 
 
+def get_nph_slack_message_handler() -> SlackMessageHandler:
+    slack_config = config.getSettingJson(config.NPH_SLACK_WEBHOOKS, {})
+    if slack_config is None:
+        logging.warning("'slack_config' for 'NPH_SLACK_WEBHOOKS' is empty")
+
+    webhook_url = slack_config.get('rdr_nph_alerts', None)
+    if webhook_url is None:
+        logging.warning("'rdr_nph_alerts' is not available in slack config. 'webhook_url' is None")
+    return SlackMessageHandler(webhook_url=webhook_url)
+
+
```

Comment:
This needs to be moved to a cloud task call that we use for all other NPH incidents.  `NphIncidentTaskApi`